hrjobcontract/templates/CRM/Hrjobcontract/Import/Form/Preview.tpl

```diff
@@ -91,7 +91,7 @@
 
 
  {* Table for mapping preview *}
- {include file="CRM/Contact/Import/Form/MapTable.tpl}
+ {include file="CRM/Hrjobcontract/Import/Form/MapTable.tpl"}
```

Comment:
This is how it should be ... This file should be taken from the extension template file . (but anyway I  don't really think it does matter  because they are probably similar )